EntryAutoComplete/EntryAutoComplete/EntryAutoComplete/CustomControl/EntryAutoComplete.cs

```diff
@@ -204,14 +211,12 @@ private void InitClearImage()
 
         private void InitSuggestionsListView()
         {
-            SuggestionsListView = new ListView
+            SuggestionsStackLayout = new StackLayout
```

Comment:
It's a good solution because stack layout is a container which can automatically adjust a height.